tgui/packages/tgui/interfaces/CharacterPreferences/GameSettingsTab.tsx

```diff
@@ -0,0 +1,274 @@
+          </Button>
+        </LabeledList.Item>
+        <LabeledList.Item label="Messages">
+          <Box mb="5px" color="label">
+            Toggles if certain messages are shown in the chat window by default.
+            You can change these mid-round by using the Toggle OOC/LOOC commands
+            under the Commands tab in the top right.
+          </Box>
+          {data.isMentor ? (
+            <Box mb="5px">
+              <Button.Checkbox
+                checked={data.seeMentorPms}
+                onClick={() => act("update", { seeMentorPms: 1 })}
+              >
+                Display Mentorhelp
+              </Button.Checkbox>
+            </Box>
+          ) : null}
+          <Box mb="5px">
+            <Button.Checkbox
+              checked={data.listenOoc}
+              onClick={() => act("update", { listenOoc: 1 })}
+              tooltip="Out-of-Character chat. This mostly just shows up on the RP server and at the end of rounds."
+            >
+              Display OOC chat
+            </Button.Checkbox>
+          </Box>
+          <Box mb="5px">
+            <Button.Checkbox
+              checked={data.listenLooc}
+              onClick={() => act("update", { listenLooc: 1 })}
+              tooltip="Local Out-of-Character is OOC chat, but only appears for nearby players. This is basically only used on the RP server."
+            >
+              Display LOOC chat
+            </Button.Checkbox>
+          </Box>
+          <Box mb="5px">
+            <Button.Checkbox
+              checked={!data.flyingChatHidden}
+              onClick={() => act("update", { flyingChatHidden: 1 })}
+              tooltip="Chat messages will appear over characters as they're talking."
+            >
+              See chat above people&apos;s heads
+            </Button.Checkbox>
+          </Box>
+          <Box mb="5px">
+            <Button.Checkbox
+              checked={data.autoCapitalization}
+              onClick={() => act("update", { autoCapitalization: 1 })}
+              tooltip="Chat messages you send will be automatically capitalized."
+            >
+              Auto-capitalize your messages
+            </Button.Checkbox>
+          </Box>
+          <Box mb="5px">
+            <Button.Checkbox
+              checked={data.localDeadchat}
+              onClick={() => act("update", { localDeadchat: 1 })}
+              tooltip="You'll only hear chat messages from living people on your screen as a ghost."
+            >
+              Local ghost hearing
+            </Button.Checkbox>
+          </Box>
+        </LabeledList.Item>
+        <LabeledList.Item label="HUD Theme">
+          <Box mb="5px">
+            <Button onClick={() => act("update", { hudTheme: 1 })}>
+              Change
+            </Button>
+          </Box>
+          <Box>
+            <img
+              style={{
+                "-ms-interpolation-mode": "nearest-neighbor",
+              }}
```

Comment:
Hey, we have a lot of images that use `-ms-interpolation-mode`, it is a very common thing, it would be great to make a generic `Image` component that inherits `Box` properties, and adds a `pixelated` shorthand property.

```jsx
<Image
  pixelated
  width="32px"
  height="32px"
  src={`hud_preview_${data.hudTheme}.png`} />
```

This could use a CSS class, e.g. `Image--pixelated`, this would be a bit more performant than inline styles.